### ISSUE TITLE
Cannot load saves on Android

### ISSUE THREAD

**Issue by QuentinMcCarthy** [Ormael13/CoCX#652]
Recently discovered this mod at random and am loving it considerably so far! Very extensive.

I'm currently experiencing an issue while trying to get the game going on my Android phone. The game seems to run fine - though I've yet to move past the main menu - but if I try to load a save file the text window goes blank and nothing happens. I get no prompt to select a file or anything. Quite devastating, as I'd really rather not have to run two separate games between PC and mobile, particularly as I've already done a good bit of work on my current game on PC that I'd rather not have to do over again (The start of the game is admittedly kind-of slow)

**Comment by Jtecx** [comment]
Fyi, and apologies for this ancient necro, but this bug we have resolved about a year or so ago, in case you were still waiting. 

It was to do with the newer android security permissions not playing nicely with our ancient build process, the sandboxing causing us to lose access to the file system.

**Comment by QuentinMcCarthy** [comment]
Heck, I gave up on this ngl. ty for the response, though, is this even still being worked on?

**Comment by Jtecx** [comment]
> Heck, I gave up on this ngl. ty for the response, though, is this even still being worked on?

Yep, near daily updates. The main branch is out of date but the master-wip keeps churning.

We also upload the apks that are now autobuilt per repo update on discord, if you'd like to take a look once again 😁 

https://discord.gg/ZzmMjjT

Check the #trials_of_endless_stars channel. APKs are on the last pinned link and the message is itself constantly updated to keep up with the latest version for desktops, despite the age of the message itself.

**Comment by QuentinMcCarthy** [comment]
Are they no longer released here? Hm.

**Comment by Jtecx** [comment]
> Are they no longer released here? Hm.

They sort of are still, but you'd need a github account first, then you can just download from our Actions. 

Blame Github for not allowing downloading files without an account. We did consider having it auto publish each as a release but it'd quickly clog up the releases menu, and there didn't seem to be any semi-decent way to handle automatically clearing/removing older builds.

**Comment by QuentinMcCarthy** [comment]
Oh, actions! I was looking at releases which is like, 2024. Sick.
